ext/fluxnet_simulations/forcing.jl

```diff
@@ -345,23 +399,38 @@ end
      get_comparison_data(
         site_ID,
         hour_offset_from_UTC;
-        val = -9999
+        val = -9999,
+        timestamp_end = true
 )
 
 Gets and returns the a NamedTuple with the comparison
 data for the Fluxnet `site_ID`, given its hour offset from
-UTC, and the value used to indicate missing data (`val`)
+UTC, and the value used to indicate missing data (`val`), and the
+averaging period.
+
+Fluxnet as has two timestamps associated with it: TIMESTAMP_START and TIMESTAMP_END,
```

Comment:
comparison data output at the end of the averaging window to compare with Clima diagnostics